### ISSUE TITLE
possible to rewrite in rust in monoio / glommio?

### ISSUE THREAD

**Issue by hiqsociety** [smfrpc/smf#393]
possible to rewrite in rust in monoio / glommio?

**Comment by emaxerrno** [comment]
@hiqsociety yeah. look at smfc the flatbuffers compiler and look at the codegen parts. it really should be super trivial to port to glommio - glauber is a good friend, and I think he hasn't worked on glommio since he left datadog.  but picking this up as a printer for that should be a rather mechanical translation. all the ideas apply.
